js/site.js

```diff
@@ -1,7 +1,44 @@
+class Score {
+    constructor() {
+        this.scoreValues = {};
+    }
+
+    get() {
+        let siteScores = ['A', 'B', 'C', 'D', 'F'];
+        // return corresponding score or lowest score
+        let scoreIndex = 0;
+
+        if (this.scoreValues.topCompany) scoreIndex++
+        if (this.scoreValues.noHTTPS) scoreIndex++
+
+        if (this.scoreValues.totalBlocked)
+            scoreIndex += Math.ceil(this.scoreValues.totalBlocked / 10)
```

Comment:
is a container necessary? could this be `this.totalBlocked` along with the others - `topCompany`,  `noHTTPS`?